docs/reference/api.rst

```diff
@@ -366,6 +371,22 @@ Rest API
 
 The rest API is also available as a `swagger spec <openapi.html#http://>`_
```

Comment:
I think this entire thing is in the wrong place. It is now several 1000 lines deep in the Programmatic API reference. I think this should be its own top level item under references.